code-rules.tex

```diff
@@ -1,5 +1,6 @@
 \documentclass[10pt,letterpaper]{article}
 \input{plos-settings}
+\hypersetup{colorlinks=true,allcolors=blue}
```

Comment:
Not sure we can do this - the PLOS template is there for a reason.

Reply:
removed